webapp/src/shared/components/ChatBubble/ChatBubble.css

```diff
@@ -0,0 +1,31 @@
+.chat-bubble {
+  display: flex;
+  align-items: flex-end;
+}
+
+.chat-bubble-self {
+  flex-direction: row-reverse;
+}
+
+.chat-bubble-other {
+  flex-direction: row;
+}
+
+.chat-bubble-text {
+  border-radius: 8px;
+  padding: 0.625rem;
+}
+
+.chat-bubble-text > p {
+  margin: 0;
+}
+
+.chat-bubble-text-self {
+  background-color: #595959;
```

Comment:
I would give names to these colors (another variable with yet another name, in .root)

Reply:
> I think the name of the author should be in a distinct style, or even outside of the bubble. ![Screenshot from 2022-09-17 03-02-14](https://user-images.githubusercontent.com/2774420/190832008-8dbf4b75-9951-49aa-aca4-805a88fa9189.png) like this: ![chat_bubble_component_author](https://user-images.githubusercontent.com/2774420/190832427-6d9773ff-a0d9-4163-a7f2-32090d9cb876.png)
> 
> But maybe it's too bothersome. What do you people think?
> 
> EDIT: Mostly because otherwise there is no way to validate that a user actually sent a message. They may impersonate other users.

I followed the groups in whatsapp, but without changing the color of the name. Whatsapp assigns a color to every member name of the group. And I don't mind changing the style or putting the name outside the bubble.

But I can not understand how someone can impersonate other users if the name it put by the chat, not by the user.

Reply:
I've change it using the color names we already have, `--color-dark` and `--color-light-variant`. The second one is not exactly the same gray but for me it's ok. Please have a look.

Reply:
I think we should stick to the color of the [palette](https://www.figma.com/file/0es84Oh1roPphFOd2GYysn/Library) we have, so agree with using already defined vars.

Reply:
I'd say to use purple color for "SELF" messages. It's vivid and extends "funky" style of the app. It makes good contrast. I've been looking in some chat apps and they same to share the same principle of using one of the most representative colors for self messages:
<img width="624" alt="Screen Shot 2022-09-18 at 7 16 48 PM" src="https://user-images.githubusercontent.com/57688116/190920026-fc376bae-00ed-4d63-a550-b916949092e7.png">